react-components/select/step5.md

```diff
@@ -0,0 +1,67 @@
+Another customization option, available for all variants, is custom display and data for the Select options.
+
+Let's begin with customizing the data. 
+
+To change what data is passed around, and back to you on selection, you must implement a class that contains any data you wish to persist and a `toString` function, which converts whatever data you include in the structure to a string format. This allows the component to internally compare options for filtering, and gives some limited customization for the option's display.
+
+1) **Copy the following `CustomDataState` class into the `App.js` file, outside of the App class:**
+
+<pre class="file" data-target="clipboard">
+  class CustomDataState {
+    constructor(name, abbreviation, capital, founded) {
+      this.name = name;
+      this.abbreviation = abbreviation;
+      this.capital = capital;
+      this.founded = founded;
+    }
+    toString = () =>
+      `${this.name} (${this.abbreviation}) - Founded: ${this.founded}`;
+  }
+</pre>
+
+<strong>Note:</strong> This class may contain any properties you wish to maintain as data when an option is selected. The `toString` function tells the select component how to display and compare the data as a string.
+
+<strong>Note:</strong> You may import the `SelectOptionObject` interface from @patternfly/react-core if using TypeScript. This interface would enforce the required `toString` function for `CustomDataState`.
+
+2) **Replace the state's `options` property, located in the App class constructor, with the following array that uses `CustomDataState`:**
```

Comment:
Again multiple steps in the one action. Split up into overall action, and then subsequent instructions.